Client/src/Unified/Services/Submitter/BatchUntilInactiveBlock.cs

```diff
@@ -0,0 +1,249 @@
+  /// <returns></returns>
+  public IDisposable LinkTo(ITargetBlock<T[]>   target,
+                            DataflowLinkOptions linkOptions)
+    => timeoutTransformBlock_.LinkTo(target,
+                                     linkOptions);
+
+  /// <summary>
+  ///   To signal to the propagator object that a message is coming.
+  ///   Check if the pipeline is opened and the message is accepted
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="messageValue"></param>
+  /// <param name="source"></param>
+  /// <param name="consumeToAccept"></param>
+  /// <returns></returns>
+  public DataflowMessageStatus OfferMessage(DataflowMessageHeader messageHeader,
+                                            T                     messageValue,
+                                            ISourceBlock<T>       source,
+                                            bool                  consumeToAccept)
+  {
+    var offerResult = ((ITargetBlock<T>)source_).OfferMessage(messageHeader,
+                                                              messageValue,
+                                                              source,
+                                                              consumeToAccept);
+
+    if (offerResult == DataflowMessageStatus.Accepted)
+    {
+      timer_.Change(Timeout,
+                    System.Threading.Timeout.InfiniteTimeSpan);
+    }
+
+    return offerResult;
+  }
+
+  /// <summary>
+  ///   Called by a linked ITargetBlock TInput  to accept and consume a
+  ///   DataFlowMessageHeader previously offered by this ISourceBlockTOutput .
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="target"></param>
+  /// <param name="messageConsumed"></param>
+  /// <returns></returns>
+  public T[]? ConsumeMessage(DataflowMessageHeader messageHeader,
+                             ITargetBlock<T[]?>    target,
+                             out bool              messageConsumed)
+    => ((ISourceBlock<T[]?>)source_).ConsumeMessage(messageHeader,
+                                                    target,
+                                                    out messageConsumed);
+
+  /// <summary>
+  ///   Called by a linked ITargetBlock TInput  to reserve a previously offered
+  ///   DataFlowMessageHeader by this ISourceBlockTOutput .
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="target"></param>
+  /// <returns></returns>
+  public bool ReserveMessage(DataflowMessageHeader messageHeader,
+                             ITargetBlock<T[]>     target)
+    => ((ISourceBlock<T[]>)source_).ReserveMessage(messageHeader,
+                                                   target);
+
+  /// <summary>
+  ///   Called by a linked ITargetBlock  TInput  to release a previously reserved DataflowMessageHeader by this
+  ///   ISourceBlock;.
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="target"></param>
```

Comment:
You need a description for all the params
Maybe you can use directly the following:
```c#
/// <inherit_doc/>
```